apps/desktop-wallet/src/features/transactionsDisplay/transactionRow/OtherAmounts.tsx

```diff
@@ -0,0 +1,69 @@
+/*
+Copyright 2018 - 2024 The Alephium Authors
+This file is part of the alephium project.
+
+The library is free software: you can redistribute it and/or modify
+it under the terms of the GNU Lesser General Public License as published by
+the Free Software Foundation, either version 3 of the License, or
+(at your option) any later version.
+
+The library is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+GNU Lesser General Public License for more details.
+
+You should have received a copy of the GNU Lesser General Public License
+along with the library. If not, see <http://www.gnu.org/licenses/>.
+*/
+
+import { useMemo } from 'react'
+import { useTranslation } from 'react-i18next'
+
+import Amount from '@/components/Amount'
+import { TransactionRowProps } from '@/features/transactionsDisplay/transactionRow/types'
+import useTransactionTokens from '@/features/transactionsDisplay/useTransactionTokens'
+import { useTransactionInfoType } from '@/utils/transactions'
+
+interface TransactionOtherTokenAmountsProps extends TransactionRowProps {
```

Comment:
Linked to my previous comment : if we indeed go for a dedicated types.ts file, should this be declared there?

Reply:
I believe I only put in types.ts the types that are shared. I do prefer to keep the types close to the component (as long as we do not duplicate them)